solr/solr-ref-guide/modules/deployment-guide/pages/docker-faq.adoc

```diff
@@ -48,8 +48,8 @@ This is useful if you want to inspect or modify the data in the Docker host
 when the container is not running, and later easily run new containers against that data.
 This is indeed possible, but there are a few gotchas.
 
-Solr stores its core data in the `server/solr` directory, in sub-directories for each core.
-The `server/solr` directory also contains configuration files that are part of the Solr distribution.
+Solr stores its core data in the `var/solr/data` directory, in sub-directories for each core.
+The `var/solr/data` directory also contains configuration files that are part of the Solr distribution.
```

Comment:
I'm not sure this is as true for `/var/solr/data` as it was for `server/solr`. So just remove this line?
```suggestion
```